package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
```

Comment:
This is a rather personal preference than adding any benefit. I'd stay away from enforcing this as a rule (it shouldn't be linted), as I'm oftentimes copy pasting colors, and if they're provided in a short format, I don't want the devs to need to hit 3 keys without any benefit than a visual one. 

I have the same feeling about the quotes - it doesn't add value, but I agree that it's easier to type a ' than a " so at least it has some benefits.